index.js

```diff
@@ -19,38 +21,31 @@ app.use(cors());
 app.use(logger('dev'));
 app.use(express.urlencoded({ extended: false }));
 app.use(express.json());
-app.use('/api/v1/docs', swaggerUi.serve, swaggerUi.setup(docs));
 
 app.use(methodOverride());
 app.use(express.static(`${__dirname}/public`));
 
-app.use(
-  session({
-    secret: 'authorshaven',
-    cookie: { maxAge: 60000 },
-    resave: false,
-    saveUninitialized: false
-  })
-);
+app.use('/api/v1', routes);
+app.use('/api/v1/docs', swaggerUi.serve, swaggerUi.setup(docs));
 
 if (!isProduction) {
   app.use(errorhandler());
 }
 
-// catch 404 and forward to error handler
+// / catch 404 and forward to error handler
 app.use((req, res, next) => {
   const err = new Error('Not Found');
   err.status = 404;
   next(err);
 });
 
-// error handlers
+// / error handlers
 
 // development error handler
 // will print stacktrace
 if (!isProduction) {
   app.use((err, req, res, next) => {
-    console.log(err.stack);
+    // console.log(err.stack);
```

Comment:
Why not just delete this if it is no more needed?

Reply:
Okay, noted